projects/packages/jetpack-mu-wpcom/src/features/wpcom-media/wpcom-media-url-upload-form/style.scss

```diff
@@ -1,15 +1,36 @@
 .wpcom-media-url-upload-form {
-	margin-top: -15px;
-	margin-bottom: 15px;
-	height: 30px;
+	margin: -15px 30px 0;
+	font-size: 12px;
 
-	button#{&}__pre-upload-button {
-		padding: 0 21px;
+	form {
+		display: flex;
+		align-items: flex-start;
+		justify-content: center;
+		gap: 6px;
+		flex-wrap: wrap;
+		margin: 12px 0 6px;
 	}
 
 	input {
-		width: 100%;
-		max-width: 600px;
-		margin-right: 6px;
+		flex: 1;
+		height: 30px;
+		max-width: 300px;
+	}
+}
+
+#plupload-upload-ui.has-wpcom-media-url-upload {
```

Comment:
I wonder why we need `.has-wpcom-media-url-upload`? As far as I can tell, the `#plupload-upload-ui` is only present in `media-new.php` 🤔

Reply:
I just want to keep it safe to apply styles to #plupload-upload-ui only when it has `.has-wpcom-media-url-upload`. Maybe it's overthinking 🤔